app/role.go

```diff
@@ -11,11 +11,24 @@ import (
 )
 
 var (
-	accountActionGroups   = getAccountActionGroups()
-	namespaceActionGroups = getNamespaceActionGroups()
+	userAccountActionGroups = getUserAccountActionGroups()
+	allAccountActionGroups  = getAllAccountActionGroups()
+	namespaceActionGroups   = getNamespaceActionGroups()
 )
 
-func getAccountActionGroups() []string {
+func getUserAccountActionGroups() []string {
+	var rv []string
+	for n, v := range auth.AccountActionGroup_value {
+		if v != int32(auth.ACCOUNT_ACTION_GROUP_UNSPECIFIED) &&
+			v != int32(auth.ACCOUNT_ACTION_GROUP_METRICS_READ) {
```

Comment:
can a user not have the metrics read role?

Reply:
No, we prevent that and this prevents tcld from trying.